src/app/callbacks.js

```diff
@@ -232,13 +232,14 @@ createNameSpace('realityEditor.app.callbacks');
 
     /**
      * Callback for realityEditor.app.getPosesStream
-     * @param {Array<Object>} pose
+     * @param {Array<Object>} pose (in world CS, in mm units)
      * @param {number} timestamp of the pose (in miliseconds, but floating point number with nanosecond precision)
      * @param {Array<number>} [width, height] of the image which the pose was computed from
      */
     function receivePoses(pose, timestamp, imageSize) {
 
         let poseInWorld = [];
+        /*
```

Comment:
It does not seem that this commented code is necessary after the changes in the app (in particular TODO remarks below). But I want to double-check.

Reply:
If you want to make the deletion a separate commit we can refer to the diff of that if anything goes wrong in the future